libsolidity/ast/ASTAnnotations.h

```diff
@@ -142,7 +142,7 @@ struct VariableDeclarationAnnotation: DeclarationAnnotation
 	std::set<CallableDeclaration const*> baseFunctions;
 };
 
-struct StatementAnnotation: ASTAnnotation, DocumentedAnnotation
+struct StatementAnnotation: ASTAnnotation
```

Comment:
This is a rather unrelated change. But right now, we do not access the documentation annotations of statements anywhere. Therefore it was removed from the base class list

Reply:
Yes, this is a remnant of the why3 mechanism for formal verification.